### DISCUSSION TITLE
Benachrichtigung zu "Nicht verfügbare Produkte"

### DISCUSSION THREAD

**Discussion by StefanOtmar** [affiliatetheme/affiliatetheme#18] [Q&A]
- Ich habe Produkte per Amazon Import importiert.
- Weitere Shops sind als Preisvergleich ergänzt (ebay, Custom CSV).
- Da manche Produkte auf Amazon nach einiger Zeit nicht mehr angeboten werden, habe ich Amazon wieder aus dem Preisvergleich entfernt ([siehe Screenshot](https://scontent-muc2-1.xx.fbcdn.net/v/t39.30808-6/336765312_601420798218184_9160125734649860705_n.jpg?_nc_cat=111&ccb=1-7&_nc_sid=dbeb18&_nc_ohc=rGe8RP2HI4kAX9RsT6y&_nc_ht=scontent-muc2-1.xx&oh=00_AfC0KTMPjgEWfKZts0g3HoCuYKuPpqIpFWMbNTCCuuQLGw&oe=6465849B)).

Problem: Das "entfernte Produkt" ist dennoch weiterhin in der E-Mail-Benachrichtigung der "Nicht verfügbaren Produkte" enthalten.
Frage: Was muss ich tun, damit das Produkt nicht mehr in der E-Mail-Benachrichtigung enthalten ist?

Danke und Grüße
Stefan

**Comment by olaf-schmitz** [comment]
Das geht leider nicht ohne das Produkt komplett zu löschen (ich mache es genauso wie Du). Das Datum in der Mail ändert sich dann aber nicht mehr auf das tagesaktuelle Datum. Du kannst Dir also nur die Produkte anschauen mit dem tagesaktuellen Datum in der Mail und diese bearbeiten. Die ist allerdings auch eine Sache auf unserer Liste für künftige Verbesserungen.

**Comment by StefanOtmar** [comment]
Bei mir ändert sich sogar das Datum in der Mail, obwohl ich das Produkt schon vor Wochen gelöscht habe.

Aber wenn ihr das schon auf der Todo-Liste habt, ist es ja ok.

**Comment by StefanOtmar** [comment]
Ich habe das Problem mit folgendem Workaround "gelöst":
1.) Betreffendes Produkt editieren und einen neuen beliebigen Amazon Artikel importieren, welcher lieferbar ist.
2.) Speichern
3.) Den Amazon Artikel wieder entfernen.
4.) Das Produkt ist in der E-Mail Benachrichtigung nicht mehr aufgeführt.
